09-Multiparticle-Collision-Dynamics/02-Diffusion-in-Solution.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        snapshot = hoomd.Snapshot().from_gsd_frame(
</pre><p>Use <code>hoomd.Snapshot.from_gsd_frame</code> - it is a class method. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>